workflow/rocoto/tasks.py

```diff
@@ -11,25 +11,26 @@
 
 
 class Tasks:
-    SERVICE_TASKS = ['arch_vrfy', 'arch_tars', 'earc_vrfy', 'earc_tars', 'stage_ic', 'fetch', 'cleanup']
+    SERVICE_TASKS = ['arch_vrfy', 'earc_vrfy', 'stage_ic', 'cleanup']
+    DTN_TASKS = ['arch_tars', 'earc_tars', 'fetch']
```

Comment:
are these `DTN_TASKS` on hera/wcoss2 etc?

Reply:
Not exactly.  There's a hierarchy established in `get_resource` so that if there is a `DTN` cluster, partition, and/or queue, it will be used.  If not, then the `SERVICE` cluster, partition, and/or queue will be used.  And if there isn't a `SERVICE` cluster, partition, or queue, then the `BATCH` options are used:

https://github.com/NOAA-EMC/global-workflow/blob/f1be19006f1c59b4d3441a525388a62bd4682cc5/workflow/rocoto/tasks.py#L362-L397